units/src/amount/signed.rs

```diff
@@ -238,7 +324,7 @@ impl SignedAmount {
     ///
     /// Consider using `unsigned_abs` which is often more practical.
     ///
-    /// Returns [`None`] if overflow occurred. (`self == MIN`)
+    /// Returns [`None`] if overflow occurred. (`self == i64::MIN`)
```

Comment:
Seeing this change (which is close enough to correct) made me write #3787.

Reply:
Yeah I noticed a few things that will need to be changed soon.  I think it's clearer leaving it as `i64::MIN` for whoever addresses #3787 that this will need to be changed.